maquetado/login.html

```diff
@@ -15,8 +15,8 @@
       <input name="email" id="email" class="input-field" type="email">
       <label for="password" class="input-description">Password</label>
       <input name="password" id="password" class="input-field" type="password">
-      <button href="tbdef" class="login-detail">Log in</button>
-      <button href="tbdef" class="sign-up-detail">Sign Up</button>
+      <button href="tbdef" class="confirmation-detail">Log in</button>
```

Comment:
You should made this changes in the `login` PR. I made some reviews about this there.